modules/blackhole-monitor/src/monitor.py

```diff
@@ -0,0 +1,116 @@
+import boto3
+import logging
+import json
+import os
+
+cloudwatch = boto3.client('cloudwatch')
+ec2 = boto3.client('ec2')
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+def lambda_handler(event, context):
+  # Get transit gateways
+  getTransitGateways = ec2.describe_transit_gateways()
+
+  # Loop over transit gateways
+  for i in getTransitGateways['TransitGateways']:
```

Comment:
for loop variables should be descriptive..

e.g `for gateway in getTransitGateway`

Reply:
https://github.com/ministryofjustice/hmpps-ems-platform-terraform-modules/pull/7/commits/2c597cb6676cc92b7c42e7604a002e740e661e2b